core/sdk/src/main/java/com/mongodb/stitch/core/services/internal/CoreStitchServiceClientImpl.java

```diff
@@ -195,14 +202,44 @@ public CoreStitchServiceClient withCodecRegistry(final CodecRegistry codecRegist
 
   @Override
   public void bind(final StitchServiceBinder binder) {
-    this.serviceBinders.add(new WeakReference<>(binder));
+    this.serviceBinders.put(new WeakReference<>(binder), Boolean.TRUE);
```

Comment:
[nit] can you not just pass `true` here?

Reply:
You can't satisfy generics with primitive types in Java, so it would be auto-wrapped to the equivalent of this anyway, so it's easier to use the constant.